lib/perl/Genome/Model/Tools/Picard.pm

```diff
@@ -315,9 +336,52 @@ sub run_java_vm {
     return 1;
 }
 
+sub _collect_call_info {
```

Comment:
This sub is temporary. We're hoping to collect metrics about how Picard is called in practice before I start messing with default version handling.

I'm hoping that wrapping in an `eval` and passing a timeout to `IO::Socket::INET->new()` is sufficient protection against this sub taking down a whole build or getting stuck waiting on a socket.